Sources/SnowManLogic/GameLogic/Game.swift

```diff
@@ -14,14 +14,18 @@ public class Game {
 	}
 
 	public func playGame() {
+		let maxNumberOfGuesses = guessManager.maxNumberOfGuesses
+		view.promptNumberOfGuesses()
+		let numberOfGuesses = view.receiveNumberOfGuesses(maxNumberOfGuesses)
```

Comment:
This is going to break a lot fewer things if I can merge my other branch before I make these changes. Is there something that I can do to make that happen @mnewsome @zacharyjdavy